### DISCUSSION TITLE
Is there any way to create label by specifing the x, y, and content?

### DISCUSSION THREAD

**Discussion by ToeJammm** [recharts/recharts#4716] [Q&A] [answered]
I am making a graph where I want to allow users to place there own labels after clicking somewhere on the graph. I have the access to all of the positional data, I just don't see any API for allowing a label based on those attributes

**Comment by ckifer** [comment]
You can use `Label` with a `content` prop that returns a `<Text>` element and specify your own coordinates. You could also just render your own recharts `<Text>` or svg `<text>`

Text ref https://github.com/recharts/recharts/blob/3.x/src/component/Text.tsx#L58 (takes in all text props including x, y)

**Reply by ToeJammm** [reply]
could you post an example in a sandbox of rendering your own <text> element with x and y values please?

**Reply by ckifer** [reply, marked as answer]
[this](https://codesandbox.io/p/sandbox/line-chart-with-customized-label-forked-phnn95?file=%2Fsrc%2FApp.tsx%3A62%2C20) should do it

**Reply by ToeJammm** [reply]
these examples are x and y values based on number of pixels from the top left corner of the graph it seems. I am also wondering if there is a way to render the text positions based on the actual graph X and Y values? Not the pixel distances.

**Reply by ckifer** [reply]
No there isn't - at this point you are specifying where in the graph from 0,0 starting in the top left the text will render.

In 3.x we'll be providing hooks to be able to do data-to-pixel transformations but it won't be for a while

**Reply by ToeJammm** [reply]
one quick question...how do I turn the animations off. I have tried a few times looking at the docs and it hasn't worked for me yet.

**Reply by ckifer** [reply]
`isAnimationActive={false}` on any graphical element (Line, Bar, etc.)
https://codesandbox.io/p/sandbox/line-chart-with-customized-label-forked-phnn95?file=%2Fsrc%2FApp.tsx%3A62%2C20
